nightly_tests/marketplace_config.yaml

```diff
@@ -1,9 +1,13 @@
+ManagementConsole:
+  sha:
+  release: 24.10-stable
+
 MarketplaceItems:
   - name: unity-cs-monitoring-lambda
-    version: 0.1
+    version: 24.4-stable
```

Comment:
See my comments in slack about this type of version.  The question is whether the JS library is blowing up with "Invalid Version" due to this not being a valid semantic version identifier?